app/src/main/java/com/thoughtworks/aceleradora/solicitacao/controller/SolicitacaoController.java

```diff
@@ -52,22 +65,36 @@ public String listaSolicitacoesDoHospital(Model model) {
     }
 
     @PostMapping("/cadastro")
-    @ResponseBody
-    public String salvaSolicitacao(Model model, @ModelAttribute("solicitacao") Solicitacao solicitacao) {
+    public String salvaSolicitacao(Model model, Solicitacao solicitacao) {
         solicitacao.getAcompanhantes().forEach(acompanhante -> acompanhante.setSolicitacao(solicitacao));
-
+        solicitacao.setStatus("Pendente");
+        solicitacao.setDataAtualizacao(LocalDateTime.now());
```

Comment:
https://stackoverflow.com/questions/221611/creation-timestamp-and-last-update-timestamp-with-hibernate-and-mysql

Reply:
atualizado conforme link enviado  
  @PrePersist
    protected void onCreate() {
        dataAtualizacao = LocalDateTime.now();
    }
    @PreUpdate
    protected void onUpdate() {
        dataAtualizacao = LocalDateTime.now();
    }